agents/plugins/apache_status

```diff
@@ -162,6 +176,16 @@ def main():
     if not servers:
         return 0
 
+    overwrite_conf = {'custom': config.get("CUSTOM_ADDRESS_OVERWRITE", {})}
+    if config.get('ENABLE_OMD_SITE_DETECTION') and os.path.exists('/usr/bin/omd'):
+        for line in os.popen('omd sites').readlines():
```

Comment:
lieber subprocess benutzen. Was ist, wenn der Aufruf fehlschlägt? Wenn Du das nicht behandeln musst, schreib einen Kommentar hin, damit man bescheid weiß.

Reply:
popen benutze ich nur, weil es ansonsten auch in diesem Script verwendet wird und ich ich neues Modul einführen will.  Wenn /usr/bin/omd existiert, geht in einem normalen Fall nichts kaputt. Und wenn es hier wirklich mal einen Sonderfall geben sollte, muss man den dann behandeln. Aber ich habe aktuell keinen Fall gefunden, worst case ist immer nur eine leere Liste.